### PR TITLE
feat(minesweeper): init

### PR DESCRIPTION
**Pull request by GitMuslim**
## 🎉 Theme for Minesweeper Online 🎉

title

## 💬 Additional Comments 💬

tiny edits on other stuff in userstyles.yml file

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [Catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk#readme).

### CODE REVIEW

File: scripts/userstyles.yml
Comment (isabelroses) [inline scripts/userstyles.yml:1]: If you want to change other readme's please open another PR and leave the rest the same in this pr.

**Review by isabelroses — CHANGES_REQUESTED** [review]
Other then the comments, the css is good.

**Comment by isabelroses** [comment]
Seems the linter is throwing some errors around you setting margins etc, do we need these?

**Comment by GitMuslim** [comment]
> Seems the linter is throwing some errors around you setting margins etc, do we need these?

fixed

File: styles/minesweeper/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @color: @catppuccin[@@lookup][@@accent];
```
Comment (uncenter) [inline styles/minesweeper/catppuccin.user.css:54]: Can we avoid changing these variables? It is supposed to be `@accent-color` for consistency with all the rest of the userstyles.

Comment (GitMuslim) [reply styles/minesweeper/catppuccin.user.css:54]: yeah sorry I did not notice that, I'll open a PR to edit it

Comment (GitMuslim) [reply styles/minesweeper/catppuccin.user.css:54]: https://github.com/catppuccin/userstyles/pull/589

**Pull request merged**